test/tests/functional/pbs_qstat.py

```diff
@@ -43,6 +43,7 @@ class TestQstat(TestFunctional):
     This test suite validates output of qstat with various options
     """
 
+    @skipOnCpuSet
```

Comment:
Please check if delete and create node works fine on cpuset machine

Reply:
@shilpakodli  Deleting and creating node works fine on cpuset machine. In this particular tests where the tests(test_qstat_n_fqdn and test_qstat_n_ip) are recreating nodes the tests are failing on cpuset because in the test we are expecting the num of nodes to be free is 1 but observed count is >1(due to vnodes). 

Instead of skipping the test, I have updated it accordingly in my latest commit.